File: chapter2/initSpark.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#########################################################################
# File Name: initSpark.py
# Author: lpqiu
# mail: qlp_1018@126.com
# Created Time: Fri Nov 14 01:02:19 2014
#########################################################################
# Initializing Spark in Python
from pyspark import SparkConf,SparkContext

conf = SparkConf().setMaster("local").setAppName("My App")
sc = SparkContext(conf)

File: chapter2/initSpark.java
/*************************************************************************
    > File Name: initSpark.java
    > Author: lpqiu
    > Mail: lpqiu_1018@126.com 
    > Created Time: Fri Nov 14 02:24:15 2014
 ************************************************************************/
import org.apache.spark.SparkConf;
import org.apache.spark.api.java.JavaSparkContext;

SparkConf conf = new SparkConf().setMaster("local").setAppName("My App");
JavaSparkContext sc = new JavaSparkContext(conf);
